src/App.js

```diff
@@ -1,10 +1,176 @@
-import React from 'react';
+import React, {useState} from 'react';
 import './App.css';
+import logo from './assets/screenshot-logo.png'
+import FruitButtons from "./FruitButtons";
+import {fireEvent} from "@testing-library/react";
+import FormEntry from "./FormEntry";
 
 function App() {
+
+
+    const [fruitState, setFruitState] = useState( {
+        strawberries : 0,
+        bananas : 0,
+        apples : 0,
+        kiwis : 0
+    })
+
+    const [formState, setFormState] = useState({
+        firstname : '',
+        lastname: '',
+        age : 0,
+        postalCode : '',
+        frequency : 'Iedere week',
+        moment : 'daytime',
+        comment : '',
+        conditions : false,
+    })
+
+
+
+    function handleChange(e) {
+        const changedFieldName = e.target.name;
+        const newValue = e.target.type === "checkbox" ? e.target.checked : e.target.value;
+        setFormState({
+            ...formState,
+            [changedFieldName]: newValue,
+        });
+    }
+
+    // function logFormValues
+
+    // This function resets all the counters for the four fruits
+    function handleReset() {
+        setStrawberries(strawberries-strawberries)
+        setKiwis(kiwis-kiwis)
+        setBananas(bananas-bananas)
+        setApples(apples-apples)
+    }
+
+    function handleSubmit(e) {
+        e.preventDefault();
+        console.log(formState)
+    }
+
   return (
     <>
-      <h1>Fruitmand bezorgservice</h1>
+        <img src={logo} alt="Logo of our fruit delivery service"/>
+
+        <FruitButtons
+            title="🍓 Aardbeien"
+            buttonNameMinus="minusButton"
+                buttonNamePlus="plusButton"
+                disabled={fruitState.strawberries === 0}
+                onClickMinus={() => setFruitState({...fruitState, strawberries: fruitState.strawberries-1 })}
```

Comment:
Als je de huidige state wil gebruiken, dan kun je beter die gebruiken die meegegeven wordt
```suggestion
                onClickMinus={() => setFruitState(currentState => ({...currentState, strawberries: currentState.strawberries-1 }))}
```
Hierdoor wordt er niet telkens een nieuwe functie aangemaakt in memory, omdat een waarde buiten de functie veranderd wordt